bfrec

```diff
@@ -44,8 +44,8 @@ Usage: $PROGNAME    [--help]
 
 Description:
   --help                : print help.
-  --default [FILE]      : update the boot partition via the
-                          kernel path (deprecated). If FILE
+  --default [FILE]      : update the boot partition use default
```

Comment:
this is previously marked as "depracated". It is intended to be removed in the future. Plus I don't think "default mode" is clear enough.

Reply:
Removed.